ci/pipelines/cf-deployment.yml

```diff
@@ -1793,6 +1836,260 @@ jobs:
     - put: windows-pool
       params: {release: windows-pool}
 
+- name: fips-acquire-pool
+  serial: true
+  public: true
+  plan:
+    - in_parallel:
+        - get: cf-deployment-develop
+          trigger: true
```

Comment:
I think this is missing a `passed` section. See above for examples.

Reply:
Yes, good catch. Added "passed" jobs.